content/docs/dvclive/ml-frameworks/huggingface.md

```diff
@@ -26,31 +32,61 @@ trainer.add_callback(DVCLiveCallback(save_dvc_exp=True))
 trainer.train()
 ```
 
-Each metric will be logged to:
-
-```py
-{Live.plots_dir}/metrics/{split}/{metric}.tsv
-```
+## Parameters
 
-Where:
+- `live` - (`None` by default) - Optional [`Live`] instance. If `None`, a new
+  instance will be created using `**kwargs`.
 
-- `{Live.plots_dir}` is defined in [`Live`].
-- `{split}` can be either `train` or `eval`.
-- `{metric}` is the name provided by the framework.
+- `log_model` - (`None` by default) - use
+  [`live.log_artifact()`](/doc/dvclive/live/log_artifact) to log checkpoints
+  created by the
+  [`Trainer`](https://huggingface.co/docs/transformers/main_classes/trainer#checkpoints).
 
-## Parameters
+  - if `log_model is None` (default), no checkpoint is logged.
 
-- `model_file` - (`None` by default) - The name of the file where the model will
-  be saved at the end of each `step`.
+  - if `log_model == 'True'`, the final checkpoint is logged at the end of
+    training.
 
-- `live` - (`None` by default) - Optional [`Live`] instance. If `None`, a new
-  instance will be created using `**kwargs`.
+  - if `log_model == 'all'`, all checkpoints are logged during training.
+    [`live.log_artifact()`] is called with `Trainer.output_dir`.
```

Comment:
```suggestion
    `Live.log_artifact()` is called with `Trainer.output_dir`.
```

I think this will auto-link (currently it shows a broken link). Let's update all the `live.log_...` mentions in the page to `Live.log...` for consistency.